packages/uploadthing/src/sdk/index.ts

```diff
@@ -36,29 +40,26 @@ import type {
   UTApiOptions,
 } from "./types";
 import { UTFile } from "./ut-file";
-import {
-  downloadFiles,
-  guardServerOnly,
-  parseTimeToSeconds,
-  uploadFilesInternal,
-} from "./utils";
+import { downloadFiles, guardServerOnly, uploadFilesInternal } from "./utils";
 
 export { UTFile };
 
 export class UTApi {
   private fetch: FetchEsque;
-  private defaultHeaders: FetchContextService["baseHeaders"];
+  private defaultHeaders: Record<string, string | undefined>;
   private defaultKeyType: "fileKey" | "customId";
   private logLevel: LogLevel | undefined;
+  private token: typeof UTToken.Type;
+
   constructor(opts?: UTApiOptions) {
     // Assert some stuff
     guardServerOnly();
     incompatibleNodeGuard();
-    const apiKey = getApiKeyOrThrow(opts?.apiKey);
+    this.token = Effect.runSync(getToken(opts?.token));
 
     this.fetch = opts?.fetch ?? globalThis.fetch;
     this.defaultHeaders = {
-      "x-uploadthing-api-key": apiKey,
+      "x-uploadthing-api-key": undefined,
```

Comment:
Is this required?

Reply:
nope. fixeed:

https://github.com/pingdotgg/uploadthing/blob/30fbfd78d8319bb0043a4a93b7c3a89823a4c983/packages/uploadthing/src/sdk/index.ts#L51-L84